packages/apollo-server-core/src/__tests__/runQuery.test.ts

```diff
@@ -479,6 +479,30 @@ describe('runQuery', () => {
       });
     });
 
+    describe('didResolveSource', () => {
+      const didResolveSource = jest.fn();
+      it('called with the source', async () => {
```

Comment:
Might be nice to have a comment noting that the APQ path is tested in the integration test suite?

Reply:
Agree. 464e4f2128fcd5bad39aab22b8b7755c9e53a29f  Thanks!